policyengine_us/variables/gov/local/co/denver/property_tax_relief/denver_property_tax_relief_homeowner.py

```diff
@@ -0,0 +1,31 @@
+from policyengine_us.model_api import *
+
+
+class denver_property_tax_relief_homeowner(Variable):
```

Comment:
```suggestion
class co_de_property_tax_relief_homeowner(Variable):
```

Reply:
can we make all variable class names start with `co_de_x` this is consistent with what we have done in Los Angeles